src/templates/base/store/global/actions.js

```diff
@@ -0,0 +1 @@
+export const DUMMY_ACTION = 'DUMMY ACTION';
```

Comment:
Instead of dummy lets create one for App level Error

Reply:
done.